.drone.yml

```diff
@@ -28,101 +38,127 @@ steps:
     image: autonomy/build-container:latest
     pull: always
     environment:
-      BUILDKIT_HOST: tcp://buildkitd.ci.svc:1234
+      BUILDKIT_HOST: unix:///run/buildkit/buildkitd.sock
```

Comment:
I don’t think we need this. I had a PR the other day that just used the default path. Might be nice to not have to worry about setting this env var.

Reply:
I set this because our makefile defaults to `tcp://0.0.0.0:1234` to account for the local setup. Something to work through.